airbyte-integrations/connectors/source-mysql/src/test/resources/expected_cloud_spec.json

```diff
@@ -214,6 +214,16 @@
                 "default": "Fail sync",
                 "order": 3,
                 "always_show": true
+              },
+              "initial_load_timeout_hours": {
+                "type": "integer",
+                "title": "Initial Load Timeout in Hours (Advanced)",
+                "description": "The amount of time an initial load is allowed to continue for before catching up on CDC logs.",
+                "default": 8,
+                "min": 4,
```

Comment:
would 4 hour minimum be too long?

Reply:
I don't think so! It's hard to have binlog retention be set to so low